src/main/java/com/google/gerrit/extensions/api/changes/ChangeApi.java

```diff
@@ -182,12 +182,12 @@ SubmittedTogetherInfo submittedTogether(
   void publish() throws RestApiException;
 
   /** Rebase the current revision of a change using default options. */
-  default void rebase() throws RestApiException {
-    rebase(new RebaseInput());
+  default ChangeInfo rebase() throws RestApiException {
```

Comment:
This file (and all others in the package `com.google.gerrit`) are (almost) 1:1 copies from Gerrit source. Your adjustment makes sense to me, but I would prefer to not modify this file. Would your change still make sense (with keeping `void` return instead of `ChangeInfo`)? An alternative would be to create a change against official Gerrit for this adjustment, and once submitted, apply it also in this project.

Reply:
Yes, it still makes sense. The main intention is to implement rebase, which I wanted to use for my use case. Then I read the API docs and saw that the spec defines a return value and thought that this additional change might make sense, but it is not required for my case.

I will modify the change now and might consider reporting to Gerrit. I was not aware that I would find such code there, too.